shared/devices/device-revoke/index.js

```diff
@@ -25,15 +25,10 @@ const Header = ({name, icon}) => (
 
 const Body = ({endangeredTLFs, name, currentDevice}) => (
   <Box>
-    <Box style={styleHeader}>
-      <Text type="BodySemibold">Are you sure you want to revoke </Text>
-      {currentDevice ? (
-        <Text type="BodySemibold">your current device</Text>
-      ) : (
-        <Text type="BodySemiboldItalic">{name}</Text>
-      )}
-      <Text type="BodySemibold">?</Text>
-    </Box>
+    <Text type="BodySemibold">
```

Comment:
fix the styling of this slightly. likely not needed w/ @nathunsmitty s larger changes to this screen